includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -235,7 +243,7 @@ private function getLoggedInUserData( $user ) {
 
 		$logOutLink = [
 			'type' => 'link-authentication',
-			'href' => '/logout',
+			'href' => $this->getLogoutUrl(),
```

Comment:
Why not use `$this->getHref( 'user-logout' )` and go the same path as `register` and `sign-in`?